src/gui_tool/gui/bb/BoundingBoxManager.py

```diff
@@ -5,19 +5,24 @@
 
 
 class BoundingBoxManager(object):
-    BOX_DISPLAY = {"color": (255, 255, 255), "lineType": 1, "thickness": 1}
+    BOX_DISPLAY = {
+        "color": (255, 255, 255),
+        "lineType": cv2.LINE_AA,
+        "thickness": 2
```

Comment:
can we keep the thickness at 1? having thicker boxes makes it harder to tell where the real edge is

Reply:
That's a good point, but if I leave it at 1, then some of the sides of the box flicker while animating between frames.

Reply:
oh okay than 2 is probably better, dont want a lot of visual flickering